packages/sheet/src/placeholders.test.ts

```diff
@@ -0,0 +1,172 @@
+import { applyPlaceholders, createPlaceholders, applyT } from "./placeholders";
+import { describe, expect, it } from "vitest";
+import { UserTheme } from "./theme";
+
+describe("Theme utility functions", () => {
```

Comment:
please remove shortcuts c and b from test.

Reply:
Done